### PR TITLE
Replaced google with Profile screen

### PR DESCRIPTION
**Pull request by RSJunior37**
As per title, removed google browser and (hopefully) all its dependencies, tests included. The new profile panel currently only display selected contact and sync any changes in info. Need the insurance class and related classes up to proceed.

Please scrutinize the changes especially those under Test files, especially guitest and handles. All deletions made are intended to only affect any related URL/Browser dependencies and it is very possible I made too many changes than necessary and proofreading is extremely needed. Also, the BrowserPanelHandle (now ProfilePanelHandle) are completely rewritten and already used in the ProfilePanelTest. That requires checking as well.

### CODE REVIEW

**Comment by RSJunior37** [comment]
Right, also whoever removed the storage file (data/addressbook.xml) from gitignore, please put it back in

File: src/main/java/seedu/address/ui/ProfilePanel.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.events.ui.PersonPanelSelectionChangedEvent;
+import seedu.address.model.person.ReadOnlyPerson;
+
+/**
+ * The Profile Panel of the App.
+ */
+public class ProfilePanel extends UiPart<Region> {
+
+    public static final String DEFAULT_MESSAGE = "Ain't Nobody here but us chickens!";
```
Comment (OscarWang114) [inline src/main/java/seedu/address/ui/ProfilePanel.java:23]: 😆

File: src/test/java/seedu/address/ui/HelpWindowTest.java [hunk 2]
```diff
@@ -24,10 +18,4 @@ public void setUp() throws Exception {
         FxToolkit.showStage();
         helpWindowHandle = new HelpWindowHandle(helpWindowStage);
     }
-
-    @Test
-    public void display() {
-        URL expectedHelpPage = HelpWindow.class.getResource(USERGUIDE_FILE_PATH);
-        assertEquals(expectedHelpPage, helpWindowHandle.getLoadedUrl());
-    }
```
Comment (OscarWang114) [inline src/test/java/seedu/address/ui/HelpWindowTest.java:32]: Hmm I don't think this test should be deleted. It is testing if the UserGuide.html can be displayed properly and has nothing to do with the BrowserPanel. You can merge the PR after restoring it.

**Review by OscarWang114 — APPROVED** [review]
Great. Everything looks good except for one test case which I think should be restored.

**Comment by OscarWang114** [comment]
I have submitted a PR of the Insurance, LifeInsurance class and recovered the changes in .gitignore. Would appreciate your review!

**Pull request merged**